lib/rmt/mirror.rb

```diff
@@ -165,8 +163,10 @@ def mirror_data(primary_files, deltainfo_files)
       )
       parser.parse
       to_download = parsed_files_after_dedup(@repository_dir, parser.referenced_files)
-      @downloader.download_multi(to_download) unless to_download.empty?
+      failed_downloads += @downloader.download_multi(to_download, ignore_errors: true) unless to_download.empty?
```

Comment:
I would prefere << operator, since it does not create new object

Reply:
Ditto.